pwiz_tools/Skyline/TestFunctional/RetentionTimeFilterTest.cs

```diff
@@ -44,7 +44,6 @@ public class RetentionTimeFilterTest : AbstractFunctionalTest
     {
         private readonly string extension = ExtensionTestContext.ExtMz5;
         
-        [TestMethod, NoParallelTesting]

```

Comment:
Looks like this is removing a test. Why?

Reply:
Yikes, that was unintentional